x/tss/types/types.go

```diff
@@ -142,8 +142,8 @@ func (m MultisigInfo) GetPubKeysByValidator(val sdk.ValAddress) []ecdsa.PublicKe
 	return pubKeys
 }
 
-// GetSigKeyPairs returns list of pub key and signature pairs
-func (m MultisigInfo) GetSigKeyPairs() []exported.SigKeyPair {
+// GetTargetSigKeyPairs returns list of pub key and signature pairs
+func (m MultisigInfo) GetTargetSigKeyPairs() []exported.SigKeyPair {
```

Comment:
```suggestion
func (m MultisigInfo) GetSigKeyPairs(includeAll ...bool) []exported.SigKeyPair {
```
so that the caller has the option to choose whether or not to get all signatures or whatever is needed to meet the threshold.

Reply:
Leaving as is for now. Might be better to have a separate function for it.